app/views/schools/advice/index/_priority_table.html.erb

```diff
@@ -1,8 +1,8 @@
 <table class="table table-borderless table-sorted advice-table advice-priority-table">
   <thead>
     <tr>
-      <th colspan="2"><th>
-      <th colspan="3"><%= t('advice_pages.index.priorities.table.columns.savings') %></th>
+      <th colspan="2"></th>
+      <th colspan="3" class='text-center'><%= t('advice_pages.index.priorities.table.columns.savings') %></th>
```

Comment:
The above two lines are the only change to do with this error - the rest are erblint corrections